cmd/hubagent/options/azureoptions.go

```diff
@@ -0,0 +1,41 @@
+//
+//Copyright (c) Microsoft Corporation.
+//Licensed under the MIT license.
+
+package options
+
+import (
+	"flag"
+)
+
+// AzurePropertyCheckerOptions holds the options for the Azure property checker.
+type AzurePropertyCheckerOptions struct {
+	// isEnabled indicates whether the Azure property checker is enabled.
+	isEnabled bool
+	// computeServiceAddressWithBasePath is the address of the Azure compute service with base path.
+	computeServiceAddressWithBasePath string
```

Comment:
```suggestion
	ComputeServiceAddressWithBasePath string
```
usually the fields are public. You can refer to https://google.github.io/styleguide/go/best-practices#option-structure